bacs/kturle/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# Maintainer: Mikhail Ivchenko <ematirov@gmail.com>
+pkgname='kturtle.interpreter-git'
+pkgver=943.2fcaeef
+pkgrel=1
+pkgdesc="Kturtle customized for BACS"
+arch=('i686' 'x86_64')
+url="https://github.com/EgorMatirov/kturtle"
+license=('GPL' 'LGPL')
```

Comment:
In project it is `GPL` only. Where does `LGPL` come from?

Reply:
Copied it from here:
https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/kturtle
Looks strange anyway.